network.cpp

```diff
@@ -105,7 +105,7 @@ std::vector<weight> register_n_weight_mutations(network n, double mut_rate, doub
     return  networks_weights;
 }
 
-std::vector<weight> register_n_activation_mutations(network n, double mut_rate, std::mt19937 &rng, int repeats)
+std::vector<weight> register_n_activation_mutations(network n, double mut_rate, std::mt19937_64 &rng, int repeats)
```

Comment:
In this file just a series of quick fixes for the rng type so that is compatible with the one actually used in `simulation`